### ISSUE TITLE
Fix labels with a parenthetical pluralization

### ISSUE THREAD

**Issue by yoomlam** [department-of-veterans-affairs/caseflow#12217]
**Goal**: For accessibility, text should be clearly communicated when read by a screen reader.  
**Problem**: Words with parenthetical pluralization are not read correctly by MacOS's VoiceOver screen reader.

**Example**: As a resulting of investigating #12094, a column name in a table header may have a pluralization in parenthesis, such as "Task(s)". MacOS's VoiceOver screen reader reads it as “Task S”, which sounds like it is repeating: “Task Task”.  It is reading the word before the parentheses, followed by the word in parenthesis. Other screen readers may handle this better.

## Acceptance Criteria

Column names with parenthetical pluralization should be read clearly and concisely, such as "Tasks" or "Task".

TODO: Create another ticket to change words in other elements.

**Comment by lomky** [comment]
_[what is this chart?](https://github.com/department-of-veterans-affairs/caseflow/issues/11533)_
```bash
1 | ||||||||
2 | |||||
3 | 
5 | 
8 | 
```

Any objection to just setting this to the pulural "Tasks"? Motion Carried.
If motivated, search for other occurrences and make similar changes.

We think this change is a 1, but we'e going to spend a 2's worth of effort to look for and correct similar occurences!
